src/Microsoft.AspNetCore.Rewrite/Operands/IOperand.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.FileProviders;
+
+namespace Microsoft.AspNetCore.Rewrite.Operands
+{
+    public interface IOperand
```

Comment:
@Eilon I couldn't find any mention of abstract vs interface in https://github.com/aspnet/Home/wiki/Engineering-guidelines#coding-guidelines